README.adoc

```diff
@@ -46,7 +48,7 @@ The application that you will be working with is a `query` service. It fetches p
 include::{common-includes}/gitclone.adoc[]
 
 // =================================================================================================
-== Trying the application
+== Try the application in the testing environment
```

Comment:
can use "Trying..." because all starting word of section titles are using "...ing ..."?

Reply:
I recommended to change all the section titles to use the imperative instead of gerund (ing) form. It's a minor nit but generally a stronger more active voice that makes a direct call to action

Reply:
If change it, it applies to all the guides need to be changed and I need to bring this up to YK and Laura.

Reply:
ok- i see what you mean. I think it would be a good change to discuss but shouldn't block this from moving forward. To stay with "ing", we could use

Trying the application in the testing environment
or
Running the application in the test environment

and ignore my other similar suggestions for headings for now

Reply:
updated all instances